.gitlab-ci.yml

```
@@ -67,7 +67,7 @@ simulation_stack:
 .spack_nmodl:
   variables:
     SPACK_PACKAGE: nmodl
-    SPACK_PACKAGE_SPEC: +python
+    SPACK_PACKAGE_SPEC: +python+testing
```

```suggestion
    SPACK_PACKAGE_SPEC: +python+tests
```
now that the name is `tests`